plugins/language_v.lua

```diff
@@ -12,6 +12,8 @@ syntax.add {
     { pattern = { '"', '"', '\\' },         type = "string"   },
     { pattern = { "'", "'", '\\' },         type = "string"   },
     { pattern = { "`", "`", '\\' },         type = "string"   },
+    { pattern = "r()'.*'",                  type = { "normal", "string" } }, -- raw string single quotes
+    { pattern = "r()\".*\"",                type = { "normal", "string" } }, -- raw string double quotes
```

Comment:
Do raw strings support multiline?
If so I'd use something like
```suggestion
    { pattern = { "r'", "'" },              type = "string" }, -- raw string single quotes
    { pattern = { 'r"', '"' },              type = "string" }, -- raw string double quotes
```

Reply:
Yes, V raw strings support multiline. But is there a way to exclude the `r` from the string highlighting? If not, I will commit your suggestion :slightly_smiling_face:

Reply:
You could do something ~~ugly~~ like this:
```lua
    { pattern = { 'r%f["]', '%f["]' }, -- raw string double quotes
      syntax = {
        patterns = {
          { pattern = ".*", type = "string" },
        },
        symbols = {},
      },
      type = "normal"
    },
```
but other language plugins like python include the "letter" in the token https://github.com/lite-xl/lite-xl/blob/99d328cfd793420bd1890555330b71384616b290/data/plugins/language_python.lua#L11-L14

In the end just commit the version you prefer, either way is fine for me.